### PR TITLE
Make simple SpringBoot app

### PR DESCRIPTION
**Pull request by kksshh525**
I am Iron Man

### CODE REVIEW

File: suhyeon/build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly 'org.projectlombok:lombok'
```
Comment (skawhdgjs) [inline suhyeon/build.gradle]: It is in rootProject. build.gradle

**Comment by skawhdgjs** [comment]
Let's make a diet

**Comment by kksshh525** [comment]
@skawhdgjs Complete diet!

File: suhyeon/build.gradle [hunk 2]
```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java'
+}
+
```
Comment (skawhdgjs) [inline suhyeon/build.gradle:4]:
```
subprojects {
	apply plugin: 'java'
	apply plugin: 'io.spring.dependency-management'
	apply plugin: 'org.springframework.boot'

	group = 'com.study'
	version = '0.0.1-SNAPSHOT'
	sourceCompatibility = '1.8'
	// sourceCompatibility 랑 defualt로 같다
	targetCompatibility = '1.8'

	repositories {
		mavenCentral()
	}

	dependencies {
		compile 'org.springframework.boot:spring-boot-starter-web'
		compileOnly 'org.projectlombok:lombok'
		annotationProcessor 'org.projectlombok:lombok'
		testImplementation 'org.springframework.boot:spring-boot-starter-test'
	}
}
```
This settings are applied to this build.gradle

File: suhyeon/src/main/java/com/suhyeon/boot/Controller/MainController.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.suhyeon.boot.Controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MainController {
+    @RequestMapping("/")
+    public String home () {
+        return "I am Iron Man.";
```
Comment (skawhdgjs) [inline suhyeon/src/main/java/com/suhyeon/boot/Controller/MainController.java:10]: Gesus

**Review by skawhdgjs — APPROVED** [review]

**Pull request merged**